Lib/fontTools/cffLib/__init__.py

```diff
@@ -18,47 +18,66 @@
 # mute cffLib debug messages when running ttx in verbose mode
 DEBUG = logging.DEBUG - 1
 log = logging.getLogger(__name__)
-isCFF2 = False
 
 cffHeaderFormat = """
 	major:   B
 	minor:   B
 	hdrSize: B
 """
 
-defaultMaxStack = 193
-maxStackLimit = 513
+maxStackLimit = 513 
+# maxstack operator has been deprecated. max stack is now always 513.
+
+class CFFContext(object):
```

Comment:
I'm thinking you could even go and use metaprogramming to create two different classes for CFF1 and CFF2. They would be different objects, but generated from the same code.